client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts

```diff
@@ -231,60 +233,71 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
         });
         if(Boolean(coronaTestDate)) {
             const earliestDateToInvestigate = getDatesToInvestigate(doesHaveSymptoms, symptomsStartDate, coronaTestDate)[0];
-            deleteIrrelevantEventsLogger.info('Sending to server date to delete contact events by', Severity.LOW);
-            axios.delete('/intersections/deleteContactEventsByDate', {params: {earliestDateToInvestigate}}).then((result) => {
-                if(result.data?.data?.deleteContactEventsBeforeDate) {
-                    deleteIrrelevantEventsLogger.info('Deleting contact events finished with success', Severity.LOW);
-                } else {
-                    deleteIrrelevantEventsLogger.error(`Deleting contact events finished with errors: ${result.data}`, Severity.LOW);
+            if(Boolean(earliestDateToInvestigate)) {
```

Comment:
Check the array length first and then extract the "[0]" element inside the if condition.

Reply:
fixed